_includes/layouts/base.html

```diff
@@ -1,3 +1,6 @@
+---
+date: git Last Modified
+---
```

Comment:
Add this frontmatter to the base layout so that all pages inherit it unless they override it. This will set each page's date to the last time it was modified in git. See [11ty documentation](https://www.11ty.dev/docs/dates/) for more.